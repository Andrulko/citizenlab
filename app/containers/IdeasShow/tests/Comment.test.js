describe('<Comment />', () => {
  it('should have tests in the future', () => {
    expect(true).toEqual(true);
  });
});
