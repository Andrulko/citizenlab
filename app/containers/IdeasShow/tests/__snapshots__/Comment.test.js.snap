// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comment /> it should render nested comments indented 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Comment
    className="hsLhYK"
    commentContent={
        Object {
            "en": "anything",
          }
    }
    createAt={null}
    modifiedAt={null}
/>,
  "nodes": Array [
    <Comment
      className="hsLhYK"
      commentContent={
            Object {
                  "en": "anything",
                }
      }
      createAt={null}
      modifiedAt={null}
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Styled(undefined)
        commentContent={
                Object {
                        "en": "anything",
                      }
        }
        createAt={null}
        modifiedAt={null}
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {
          "__styled-components__": undefined,
        },
        "props": Object {
          "commentContent": Object {
            "en": "anything",
          },
          "createAt": null,
          "modifiedAt": null,
        },
        "refs": Object {},
        "state": Object {
          "generatedClassName": "hsLhYK",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Comment
          className="hsLhYK"
          commentContent={
                    Object {
                              "en": "anything",
                            }
          }
          createAt={null}
          modifiedAt={null}
/>,
        "_debugID": 2,
        "_renderedOutput": <Comment
          className="hsLhYK"
          commentContent={
                    Object {
                              "en": "anything",
                            }
          }
          createAt={null}
          modifiedAt={null}
/>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Styled(undefined)
    commentContent={
        Object {
            "en": "anything",
          }
    }
    createAt={null}
    modifiedAt={null}
/>,
}
`;
